roles/installer/defaults/main.yml

```diff
@@ -443,6 +443,9 @@ ldap_password_secret: ''
 # Secret to lookup that provides the custom CA trusted bundle
 bundle_cacert_secret: ''
 
+# Set false for basic install without operator
+update_status: true
```

Comment:
It would be nice to add and example of how to use this to the debugging docs that cover how to use this flow.  
* https://github.com/ansible/awx-operator/blob/devel/docs/troubleshooting/debugging.md#iterating-on-the-installer-without-deploying-the-operator

Could you please update that @chicuongha ?  Otherwise this change looks great, thanks!

Reply:
documentation file updated